src/actions/index.js

```diff
@@ -0,0 +1,86 @@
+export const GET_MOVIE_LIST = 'GET_MOVIE_LIST';
```

Comment:
I would recommend avoid writing code in `index` files. There is a better practice to use `index` to re-export your code